### DISCUSSION TITLE
Access Issue to Admin Page in Self-hosted AFFiNE

### DISCUSSION THREAD

**Discussion by zetstream33** [toeverything/AFFiNE#8209] [Q&A]
![image](https://github.com/user-attachments/assets/522d634f-3cc6-421f-bf3d-92c13e78ebd8)

Hello, I’d like to start by thanking everyone for their continuous support.

I am using the self-hosted version of AFFiNE. When trying to access the admin page via mydomain.com/admin, I encounter the issue shown in the attached image, where I’m unable to access the admin page due to not being recognized as an admin. The account I’m using is the same one I registered with during the initial setup of AFFiNE. Even though I correctly set the ADMIN EMAIL and PASSWORD in the ENV values when running Docker, I am still experiencing the same issue. When I checked through Portainer, I could confirm that the environment variables were entered correctly as I had written.

Could this be due to the admin page’s frontend being implemented but not yet fully functional? I’m curious if others using AFFiNE in a self-hosted environment are experiencing the same issue. Thank you.

**Comment by linkh2010** [comment]
did you modify the user_features table? it happened to me when I updated a record in that table by mistake. make sure your user has the feature_id set to 7.

**Reply by zetstream33** [reply]
This works!!! Thanks bro.
